src/SprykerEco/Yves/CrefoPay/Theme/default/views/direct-debit/direct-debit.twig

```diff
@@ -23,5 +23,25 @@
             endpointUrl: subFormVars.secureFieldsApiEndpoint,
             placeholders: subFormPlaceholders
         },
+        embed: {
+            fields: crefopayFields,
+            context: _self
+        }
     } only %}
+        {% block requestForm %}
+            <div class="is-hidden">
+                {{ form_widget(data.paymentMethodSubForm) }}
+            </div>
+            {% for field in embed.fields %}
+                <div class="spacing-y">
+                    <label class="label label--required">{{ field.name | trans }}</label>
+                    <input class="input input--expand" type="text" data-crefopay="{{ field.attribute }}" placeholder="{{ field.placeholder | trans }}">
+                </div>
+            {% endfor %}
+
+            <div class="{{ config.name }}__error {{ config.jsName }}__error spacing-y is-hidden">
+                {{ 'crefopay.required_notification' | trans }}
+            </div>
+        {% endblock %}
```

Comment:
Please adjust the `crefopay-payment-form` molecule instead of overriding the whole block.
Why the data is different, maybe it should be fixed on BE side to be compatible with other usages?

Reply:
Reduced embedded code. Input field can't be fixed from BE side, original template has another layout without inputs.